docs/index.html

```diff
@@ -0,0 +1,2 @@
+<!-- Redirect to latest version -->
+<meta HTTP-EQUIV="REFRESH" content="0; url=./1.4.1/index.html">
```

Comment:
So we'd just update this version instead of the link in the readme in the future.

Reply:
should we call this out in the comment where we update version to do it here instead of the readme?

Reply:
yes

Reply:
Probably want to make sure the process doc on Android releases is up to date as well.

Reply:
I Will keep wiki synced